shell/plugins/dashboard-store/mutations.js

```diff
@@ -51,6 +51,22 @@ export function load(state, { data, ctx, existing }) {
 
   cache.generation++;
 
+  /**
```

Comment:
All steve specific stuff should be kept in the `shell/plugins/steve` store (which sucks in getters/action/mutations from the `dashboard-store`). 

I thought we might have to do something like override the `load` fn in steve's mutations, massage data and then send on to the dashboard-store's load fn (this keeps it all pretty neat)... but stumbled on the `cleanResource` getter. That looks ideal for this use case and would just need to massage data --> existing and return that... everything else is then handled as normal

Reply:
I had to take a step back and look over how the steve and dashboard store plugins actually work but I get what you're saying now. I moved the count-updating logic to the steve `cleanResource` getter (and updated my api self-links PR similarly).